src/testing/loadtest-azure/scripts/loadtest-run-harness.ps1

```diff
@@ -0,0 +1,15 @@
+# loadtest-run-harness.ps1
+$apiEndpoint = "5e241573-3a5f-4361-bf53-1ae7bde73cb7.neu.cnt-prod.loadtesting.azure.com" #needs to be updated... will be returned when resource is created
```

Comment:
cleanup

Reply:
Cleaned them up a bit. I'd like to get rid of all `*-harness.ps1` at some point. But at the moment the add a lot of value.